metrics/testdata/prometheus_metrics

```diff
@@ -13,6 +13,9 @@ container_cpu_usage_seconds_total{cpu="cpu03",id="testcontainer",image="test",na
 # HELP container_cpu_user_seconds_total Cumulative user cpu time consumed in seconds.
 # TYPE container_cpu_user_seconds_total counter
 container_cpu_user_seconds_total{id="testcontainer",image="test",name="testcontaineralias"} 6e-09
+# container_spec_cpu_quota HELP CPU quota of the container
```

Comment:
Check the format of the lines - `HELP` should be at the start of the comment. You can run tests locally btw with `make test`.